include/mata/nft/nft.hh

```diff
@@ -62,24 +62,22 @@ public:
 
 public:
     explicit Nft(Delta delta = {}, utils::SparseSet<State> initial_states = {},
-                 utils::SparseSet<State> final_states = {}, std::vector<Level> levels = {}, const Level levels_cnt = 1,
+                 utils::SparseSet<State> final_states = {}, std::vector<Level> levels = {}, const Level levels_cnt = 0,
                  Alphabet* alphabet = nullptr)
-        : mata::nfa::Nfa(std::move(delta), std::move(initial_states), std::move(final_states), alphabet),
-        levels(std::move(levels)), levels_cnt(levels_cnt) {}
-
+        : mata::nfa::Nfa(std::move(delta), std::move(initial_states), std::move(final_states), alphabet), levels(levels), levels_cnt(levels_cnt) {}
```

Comment:
The move should occur here and in the other constructor below. Otherwise, the vector of levels is copied here unnecessarily. Do you have a reason to remove it?

Reply:
It's just a mistake. Of course, the move should be here. I'll add it.